extensions/wikia/Piggyback/Piggyback_body.php

```diff
@@ -45,18 +45,17 @@ function execute( $par ) {
 }
 
 class PBHooks {
-	public static function onLoginFormAuthenticateModifyRetval( $invoker, $username, $password, &$retVal ) {
-		global $wgEnableHeliosExt;
+	public static function onLoginFormAuthenticateModifyRetval( $invoker, $username, $password, &$retVal, $authResult ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameters "$username, $password". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)